pkg/plugins/manager/signature/manifest.go

```diff
@@ -15,7 +15,10 @@ import (
 	"path/filepath"
 	"strings"
 
+	// TODO: replace deprecated `golang.org/x/crypto` package https://github.com/grafana/grafana/issues/46050
+	// nolint:staticcheck
```

Comment:
are changes to this file necessary?  Is this because other libraries were updated with the addition of gocloud.dev/blob?

Reply:
yeah - it's an indirect dependency that was updated because of the `blob`